python/test/unit/language/test_core_amd.py

```diff
@@ -3109,7 +3108,7 @@ def test_reduce_layouts(M, N, src_layout, axis, device='cuda'):
         kernel = triton.compile(f.name)
 
     rs = RandomState(17)
-    x = rs.randint(0, 4, (M, N)).astype('float32')
+    x = rs.randint(0, 1024, (M, N)).astype('float32')
```

Comment:
Larger value increases probability of test to crush if there are some issues.
Small value masks wrong reduction.